{
    "impact": {
        "cvss": [
            [
                {
                    "vectorString": "4.8/CVSS:3.0/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:N/A:N",
                    "version": "3.0"
                }
            ]
        ]
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "undertow before versions 1.4.18.SP1, 2.0.2.Final, 1.4.24.Final was found vulnerable when using Digest authentication, the server does not ensure that the value of URI in the Authorization header matches the URI in HTTP request line. This allows the attacker to cause a MITM attack and access the desired content on the server."
            }
        ]
    },
    "data_type": "CVE",
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "undertow 1.4.18.SP1"
                                        },
                                        {
                                            "version_value": " undertow 2.0.2.Final"
                                        },
                                        {
                                            "version_value": " undertow 1.4.24.Final"
                                        }
                                    ]
                                },
                                "product_name": "undertow"
                            }
                        ]
                    },
                    "vendor_name": ""
                }
            ]
        }
    },
    "data_format": "MITRE",
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-287"
                    }
                ]
            }
        ]
    },
    "data_version": "4.0",
    "references": {
        "reference_data": [
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-12196"
            }
        ]
    },
    "CVE_data_meta": {
        "ID": "CVE-2017-12196",
        "ASSIGNER": "lpardo@redhat.com"
    }
}
